src/general/OptionsMenu.cs

```diff
@@ -59,6 +59,15 @@ public class OptionsMenu : Control
     [Export]
     public NodePath ChromaticAberrationTogglePath;
 
+    [Export]
+    public NodePath CurrentGpuPath;
+
+    [Export]
+    public NodePath VideoMemPath;
```

Comment:
According to the style guide abbreviations should avoided so this variable should preferably be named `VideoMemoryPath`. There also other places that needs the adjustment where shortening is used.